ext/pdo_firebird/firebird_statement.c

```diff
@@ -886,7 +886,7 @@ static int pdo_firebird_stmt_set_attribute(pdo_stmt_t *stmt, zend_long attr, zva
 	switch (attr) {
 		default:
 			return 0;
-		case PDO_ATTR_CURSOR_NAME:
+		case PDO_ATTR_CURSOR_NAME: ;
```

Comment:
While this would work, I think it'd be more consistent and nicer if we just create a new scope.
i.e. use brackets { & } for this case.

Reply:
Agreed. This prevents variable redeclaration errors if the same variable is declared in another case, or below the switch.

Reply:
Yeah, or worse: accidentally thinking in another block that the variable may be accessed

Reply:
@nielsdos @iluuu1994 
Thanks. I have updated the commit to use brackets in the switch case.